src/services/videos/helpers/file/index.ts

```
@@ -0,0 +1,109 @@
+      const unlinkAsync = promisify(unlink);
+      unlinkAsync(localPath).catch(console.error);
+      return reject(new Error("No response body"));
+    }
+
+    (async () => {
+      try {
+        const reader = response.body?.getReader();
+
+        while (true) {
+          //@ts-ignore
+          const { done, value } = await reader?.read();
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)